Melody/Model/PlayListModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Melody.Model
+{
+    /// <summary>
+    /// This is model class for Play list
+    /// </summary>
+    public sealed class PlayListModel
+    {
+        /// <summary>
+        /// Name of the artist
+        /// </summary>
+       public string Name { get; set; }
+
+        /// <summary>
+        /// Genere of the music
```

Comment:
Please correct spelling Genere -> Genre (spelled correctly in code, misspelled in comment)

Reply:
fixed